profiling/src/profiles/booleans.rs

```diff
@@ -0,0 +1,33 @@
+use pyo3::prelude::*;
+
+pub fn exec(iterations: u64) {
+    let gil = Python::acquire_gil();
+    let py = gil.python();
+
+    let booleans = format!("[{} \"true\"]", "\"true\",".repeat(256));
```

Comment:
Is there a more idiomatic way to create a JSON string with 256 booleans?
`["true","true","true","true",..."true"]`

Reply:
I'd actually know a way in java :sweat_smile:. For rust there's afaik no better way except maybe using serde on a Vec from `[true; 256]`, but imho the current code isn't bad.

Reply:
(btw it's 257 booleans. I'm curious if there's a measurable difference between 255, 256 and 257)